boa/interfaces.py

```diff
@@ -0,0 +1,1553 @@
+import json
+
+from vyper.semantics.types import EventT
+from vyper.semantics.types.function import ContractFunctionT
+
+from boa.vyper.contract import ABIContractFactory
+
+
+class ERCToken(ABIContractFactory):
+    abi = []
+
+    def __init__(self, token_name):
```

Comment:
not sure overriding the ctor here is necessary since it duplicates `ABIContractFactory.at()` code